pkg/flow/state-error.go

```diff
@@ -72,7 +73,17 @@ func (sl *errorStateLogic) Run(ctx context.Context, engine *engine, im *instance
 		a[i] = x
 	}
 
-	err = engine.InstanceRaise(ctx, im, NewCatchableError(sl.state.Error, sl.state.Message, a...))
+	x, err := jqOne(im.data, sl.state.Message)
+	if err != nil {
+		return
```

Comment:
Have you tested this works? I'm not sure what happens in go if your init `err` in the function definition, but then create a new `err` variable inside in the function itself.

Reply:
Tested and working.